src/ProjectSystemTools/BuildLogging/Model/Frontend/IFrontendBuildTableDataSource.cs

```diff
@@ -1,22 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using Microsoft.Build.Framework;
 using Microsoft.VisualStudio.Shell.TableManager;
 
-namespace Microsoft.VisualStudio.ProjectSystem.Tools.Providers
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.Frontend
 {
-    public interface IBuildTableDataSource : ITableDataSource
+    public interface IFrontendBuildTableDataSource : ITableDataSource
     {
         ITableManager Manager { get; set; }
 
-        bool IsLogging { get; }
+        bool IsLogging();
```

Comment:
It seems like this could be left as a property. Was there a reason for the change?

Reply:
I thought it would be a similar reason to the above, but I think this one can stay as a property, since it shouldn't need to go across server to client